internal/app/service.go

```diff
@@ -245,17 +249,15 @@ func (svc *Service) LoadConfigurableFunctionPipelines() (map[string]interfaces.F
 
 	svc.targetType = nil
 
-	if svc.config.Writable.Pipeline.UseTargetTypeOfByteArray &&
-		svc.config.Writable.Pipeline.UseTargetTypeOfMetric {
-		return nil, errors.New("can not have Pipeline UseTargetTypeOfByteArray and UseTargetTypeOfMetric both set to true")
-	}
-
-	if svc.config.Writable.Pipeline.UseTargetTypeOfByteArray {
+	switch strings.TrimSpace(svc.config.Writable.Pipeline.TargetType) {
```

Comment:
This comment got lost last time... :-(

Allows for mixed case values.

```suggestion
	switch strings.ToLower(strings.TrimSpace(svc.config.Writable.Pipeline.TargetType)) {
```